### ISSUE TITLE
postgresql drop function with RETURNS TABLE fail

### ISSUE THREAD

**Issue by fjf2002** [dbeaver/dbeaver#15907]
#### System information: 
- Operating system (distribution) and version: Windoze 10
- DBeaver version: 22.0.0

#### Describe the problem you're observing:
PostgreSQL: Drop Function will fail if the function has a RETURNS TABLE clause (i. e. has out parameters).

#### Steps to reproduce, if exist:
```sql
CREATE FUNCTION x(inparam text) RETURNS TABLE(outparm text) LANGUAGE SQL AS $$SELECT '42'$$
```
Trying to delete that function in DBeaver will fail with:

_SQL-Fehler [42883]: FEHLER: Funktion public.x(text, text) existiert nicht_

I guess the solution would be to construct the DELETE statement...
- stripping out parameters
- or probably better, use support from PostgreSQL, like `pg_get_functiondef` or `pg_get_function_identity_arguments`

**Comment by uslss** [comment]
thanks for the report

**Comment by LonwoLonwo** [comment]
Hello @fjf2002 

Thanks for the bug report. 
This issue will be fixed in the next release (22.0.2): https://github.com/dbeaver/dbeaver/issues/15563
I will close this ticket as duplicate.
